magefile.go

```diff
@@ -107,7 +110,40 @@ func Build() error {
 }
 
 func Generate() {
-	mg.SerialDeps(Protobuf, mockgen.Mockgen, ControllerGen)
+	mg.SerialDeps(Protobuf, mockgen.Mockgen, ControllerGen, GenerateOTEL)
+}
+
+func GenerateOTEL() error {
```

Comment:
Can probably remove this seeing as we have it in a separate repo

Reply:
We can probably disassociate it from the generate Target and rename it to something like fetchOTELDistro, but still convenient seeing as we do require the binary for running otel in the test environment.

Or, I could add something like `goreleaser` to the sandbox repo for releases and add it those binaries to our testbin target

Reply:
https://github.com/rancher-sandbox/opni-otel-collector/pull/3

Reply:
Yeah I think fetching the binaries is the better way to go.  An alternative to attaching the binary would be using `go get` to fetch and build the binary

Reply:
I ended up adding gorelease to the sandbox, and using a testbin target, since it falls in line with other things in the test env